--- /snap/snapcraft/current/bin/scriptlet-bin/snapcraftctl.orig	2022-08-28 19:08:21.094949962 +0000
+++ /snap/snapcraft/current/bin/scriptlet-bin/snapcraftctl.new	2022-08-28 19:07:54.054910889 +0000
@@ -29,7 +29,7 @@ quote()
 # SNAPCRAFT_INTERPRETER is specified, use that one instead.
 python3_command="${SNAPCRAFT_INTERPRETER:-$(command -v python3)}"
 
-snapcraftctl_command="$python3_command -I -c '
+snapcraftctl_command="$python3_command -c '
 import snapcraft_legacy.cli.__main__
 
 # Click strips off the first arg by default, so the -c will not be passed
@@ -55,6 +55,9 @@ fi
 if [ -n "$SNAP_ARCH" ]; then
     export SNAP_ARCH="$SNAP_ARCH"
 fi
+if [ -n "$PYTHONPATH" ]; then
+    export PYTHONPATH="$PYTHONPATH"
+fi
 
 # Required so Click doesn't whine about lack of a locale
 export LC_ALL="$LC_ALL"
